app/components/report-review-warning/template.hbs

```diff
@@ -2,10 +2,19 @@
   <li class="nav-top-list-item" title="You have reports to verify">
     {{#link-to "analysis.index"
                (query-params
+                 fromDate=null
                  toDate=unverifiedReports.reportsToDate
                  reviewer=session.data.user.id
                  editable=1
                  verified=0
+                 customer=null
+                 project=null
+                 task=null
+                 user=null
+                 billingType=null
+                 costCenter=null
+                 review=""
+                 notBillable=""
```

Comment:
@anehx I'm not sure if this is the right approach to my problem since I am resetting the query params manually. Do you maybe know a better way or can we leave it like this?

Reply:
Hm looks weird but I don't have a better idea